modules/apps/commerce/commerce-inventory-service/src/main/java/com/liferay/commerce/inventory/internal/method/DefaultCommerceInventoryMethodImpl.java

```diff
@@ -150,10 +150,15 @@ public String getLabel(Locale locale) {
 	public int getStockQuantity(
 		long companyId, long commerceChannelGroupId, String sku) {
 
-		return Math.min(
```

Comment:
this looks like a big change, IDK why it was made like this and thus IDK if this is the right fix.

Reply:
This is probably correct, but needs lots of testing